master/buildbot/test/unit/test_process_buildrequestdistributor_katana.py

```diff
@@ -815,6 +814,58 @@ def test_popNextBuildToResumeShouldCheckSelectedSlave(self):
 
         self.assertEquals((slave.name, breq.id), ('slave-02', 1))
 
+    def __check_breq_status(self, breq):
+        self.assertEqual(breq.checkMerges, True)
+        self.assertEqual(breq.retries, 0)
+        self.assertEqual(breq.hasBeenMerged, False)
+
+    def test_removeBuildRequest_for_resumeBrdict(self):
+        chooser = self.brd.katanaBuildChooser
+        chooser.unclaimedBrdicts = [{'brid': 1}, {'brid': 2}]
+        chooser.resumeBrdicts = [{'brid': 3}, {'brid': 4}]
+        breq = mock.Mock()
```

Comment:
In this case, you do not need mock, you can use simple stub
```
BregClass = namedtuple('Breg', ['brdict'])
breq = BregClass(brdict={'brid': 3})
```
But please be aware and ensure you create all fields, which is used and checked in this test (as far as I see, `checkMerges`, `retries` and `hasBeenMerged` has no initialization)